torchbenchmark/util/kernels/triton_fused_attention.py

```diff
@@ -788,17 +792,18 @@ def grid_tma(META):
                 META['BLOCK_M'],
                 HEAD_DIM_Q,
                 o.element_size(),
-                o_buf.numpy(),
+                o_buf.data_ptr(),
             )
-            desc_q.copy_(q_buf, non_blocking=True)
-            desc_k.copy_(k_buf, non_blocking=True)
-            desc_v.copy_(v_buf, non_blocking=True)
-            desc_o.copy_(o_buf, non_blocking=True)
+            #desc_q.copy_(q_buf, non_blocking=True)
+            #desc_k.copy_(k_buf, non_blocking=True)
+            #desc_v.copy_(v_buf, non_blocking=True)
+            #desc_o.copy_(o_buf, non_blocking=True)
             return (triton.cdiv(q.shape[2], META["BLOCK_M"]), q.shape[0] * q.shape[1], 1)
 
         M = torch.empty((q.shape[0], q.shape[1], q.shape[2]), device=q.device, dtype=torch.float32)
         _attn_fwd_tma[grid_tma](
-            q, v, o, desc_q, desc_k, desc_v, sm_scale, M, desc_o,  #
+            #q, v, o, desc_q, desc_k, desc_v, sm_scale, M, desc_o,  #
+            q, v, o, tl.NvTmaDesc(q_buf), tl.NvTmaDesc(k_buf), tl.NvTmaDesc(v_buf), sm_scale, M, tl.NvTmaDesc(o_buf),
```

Comment:
@embg With the latest PR4498, NvTmaDesc is no longer exposed, so tl.NvTmaDesc will not work.